docs/sources/mimir/set-up/jsonnet/migrate-from-microservices-to-read-write.md

```diff
@@ -298,7 +298,7 @@ Wait the next TSDB head compaction for ingesters (2 hours).
 You must follow the shutdown ingester procedure to avoid data loss.
 {{< /admonition >}}
 
-Follow the procedure for [shutting down ingesters]({{< relref "../../manage/run-production-environment/scaling-out.md#scaling-down-ingesters-deployed-in-multiple-zones" >}}) in `ingester-zone-a`.
+Follow the procedure for [shutting down ingesters]({{< relref "../../manage/run-production-environment/scaling-out.md#scaling-down-ingesters" >}}) in `ingester-zone-a`.
```

Comment:
Is `ingester-zone-a` a setting? If so, it would be ideal to specify, i.e. "in the `ingester-zone-a` setting."

Reply:
It's not a Mimir setting exactly, but it is the standard name for the zone-A statefulset used in some of our deployment tooling/configuration such as in the Helm chart and also in the [rollout operator](https://github.com/grafana/rollout-operator).